src/main/java/uk/gov/companieshouse/chd/order/api/dto/MissingImageDeliveriesDTO.java

```diff
@@ -32,6 +32,10 @@
 	@JsonProperty("payment_reference")
 	private String paymentReference;
 
+	@NotNull
+	@JsonProperty("filing_history_type")
+	private String filingHistoryType;
+
```

Comment:
This shouldn't have tabs in it, it should use spaces instead. If the entire class is like this, we should raise the formatting as a separate Jira issue. If not, this should be re-formatted.